configs/config.novacustom_nv4x_adl

```diff
@@ -33,9 +37,11 @@ CONFIG_EDK2_FOLLOW_BGRT_SPEC=y
 CONFIG_EDK2_SD_MMC_TIMEOUT=10
 CONFIG_EDK2_ENABLE_IPXE=y
 CONFIG_EDK2_SECURE_BOOT=y
+CONFIG_EDK2_SETUP_PASSWORD=y
 CONFIG_EDK2_DASHARO_SYSTEM_FEATURES=y
 CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y
```

Comment:
`CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y` probably shouldn't be here unless we want to enable options like IOMMU, SMM BWP or vboot lock.

Reply:
@3mdeb-karolzmyslowski I do not get your comment.

This line appears in all configs: https://github.com/Dasharo/coreboot/pull/320/commits/29130ff5bf2865c9e44184643ed77c7aa3625f14

Reply:
@miczyg1 

> unless we want to enable options like IOMMU, SMM BWP or vboot lock.

This is precisely what we are enbling here, right?
1. Setup menu password
2. Network boot enable/disable
3. SMM_BWP
4. vboot lock

Reply:
Then `CONFIG_EDK2_DASHARO_SECURITY_OPTIONS` should be enabled in all configs and this is what @3mdeb-karolzmyslowski is pointing at.  Only nv4x_adl config has this enabled. You have to copy that to other configs as well @macpijan

Reply:
Got it now. I have not changed it, so as you can see for some reason it was already enabled in one config, and not in others.

@3mdeb-karolzmyslowski can you please add it then, if required?

Reply:
Probably a result of splitting some features enabling for NV4x TGL then for NV4x ADL separately and entropy rises.

Reply:
@3mdeb-karolzmyslowski ?

Please comment if you thing it can be resolved.
I can see you mentioned it was missing for all models, but you have added only for NS5X models here: e40824c48e92fd547448b97a434b440c923a13d2

Reply:
I can see all 8 configs have `CONFIG_EDK2_DASHARO_SECURITY_OPTIONS=y` so closing this discussion